lib/thread.h

```diff
@@ -240,6 +240,12 @@ extern void thread_call (struct thread *);
 extern unsigned long thread_timer_remain_second (struct thread *);
 extern struct timeval thread_timer_remain(struct thread*);
 extern int thread_should_yield (struct thread *);
+
+extern unsigned long timeval_elapsed (struct timeval a, struct timeval b);
```

Comment:
These functions are not mine and were already part of the code. 
I'm just exposing timeval_elapsed to other modules thus I can use it in isisd.c and isis_spf_delay.c 
Is that an issue ?